config/routes.rb

```diff
@@ -145,7 +145,7 @@
 
     scope :facilities, module: 'facilities' do
       resources :va, only: %i[index show], defaults: { format: :json }
-      resources :ccp, only: %i[show], defaults: { format: :json }
+      resources :ccp, only: %i[index show], defaults: { format: :json }
```

Comment:
I know it was already there, but its weird you'd need the format specified here as json since its the default for /v0

Reply:
Looks like we do it everywhere.